jujugui/static/gui/src/test/test_bundle_import_notifications.js

```diff
@@ -124,168 +124,6 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
     });
   });
 
-  describe('bundle helpers watchAll', function() {
```

Comment:
I feel like I'm missing some piece of juju knowledge here; how is this connected to legacy juju?